src/v3/bandwidthRtc.ts

```diff
@@ -471,16 +488,19 @@ class BandwidthRtc {
     };
   }
 
-  private addStreamToPublishingPeerConnection(mediaStream: MediaStream) {
+  private addStreamToPublishingPeerConnection(mediaStream: MediaStream, codecPreferences?: CodecPreferences) {
     mediaStream.getTracks().forEach((track) => {
       const transceiver = this.publishingPeerConnection!.addTransceiver(track, {
         direction: "sendonly",
         streams: [mediaStream],
       });
 
-      // Inject DTMF into one audio track in the stream
-      if (track.kind === "audio" && !this.localDtmfSenders.has(mediaStream.id)) {
-        this.localDtmfSenders.set(mediaStream.id, new DtmfSender(transceiver.sender));
```

Comment:
This doesn't really work anyway and was causing problems. Still trying to think if there's a better short-term solution.